src/DocumentEditor.ts

```diff
@@ -101,6 +101,11 @@ export class DocumentEditor implements vscode.Disposable {
 
     private async udpateDocumentToNode(node: IDocumentNode, doc: vscode.TextDocument): Promise<void> {
         const updatedDoc: {} = await node.update(JSON.parse(doc.getText()));
+        const output = util.getOutputChannel();
+        const timestamp = Math.floor(Date.now());
```

Comment:
I would prefer if this was the timestamp from the document itself, but it's not a big deal

Reply:
Mongo documents don't typically have the time-stamp field. This code is in the DocumentEditor file, which is agnostic to DocDB/Mongo. So I don't think using time-stamp from the doc would be clean code.